### PR TITLE
Update Composer screenshots

### PR DESCRIPTION
**Pull request by DelazJ**
Renames and updates screenshots with a clearer name (remove "print_composer_" at the beginning and instead of a number at the end, precise which option it's)
Updates related link
Removes list of blending modes (in map Item properties) and replaces it by a link to vector layer properties page (avoid redundancy for this kind of thing)

### CODE REVIEW

File: source/docs/user_manual/print_composer/composer_items/composer_arrow.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ For :guilabel:`SVG Marker` you can use following options.
 
 * :guilabel:`Start marker` : Choose an SVG image to draw at the beginning of the line
 * :guilabel:`End marker` : Choose an SVG image to draw at the end of the line
-* :guilabel:`Arrow head width`: Set the size of Start and/or End marker
+* :guilabel:`Arrow head width` : Set the size of Start and/or End marker
```
Comment (yjacolin) [inline source/docs/user_manual/print_composer/composer_items/composer_arrow.rst]: As far I know, there is no space before ":"  in english. Is it a workaround for a wrong interpretation of the rst parser?

Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_arrow.rst]: No wrong interpretation. It was rather to fit how others are written. All should be fixed now.

File: source/docs/user_manual/print_composer/composer_items/composer_legend.rst [hunk 2]
```diff
@@ -132,18 +133,15 @@ figure_composer_legend_5_):
...
+.. figure:: /static/user_manual/print_composer/legend_wms.png
    :align: center
 
-   WMS LegendGraphic Dialogs |nix|
+   WMS LegendGraphic and Spacing Dialogs |nix|
 
-When you have added a WMS layer and you insert a legend composer item, a request will be send to the 
-WMS server to provide a WMS legend. This Legend will only be shown if the WMS server provides the 
-GetLegendGraphic capability. The WMS legend content will be provided as a raster image.
+When you have added a WMS layer and you insert a legend composer item, a request will be send to the WMS server to provide a WMS legend. This Legend will only be shown if the WMS server provides the GetLegendGraphic capability. The WMS legend content will be provided as a raster image.
```
Comment (yjacolin) [inline source/docs/user_manual/print_composer/composer_items/composer_legend.rst]: incorrect line size (80 caracters)

File: source/docs/user_manual/print_composer/managing_composer.rst [hunk 3]
```diff
@@ -51,7 +51,7 @@ The composer manager lists in its upper part all the available print composers i
 * rename the composer: you can also rename the composer by double-clicking on its title in the upper part
 * remove the composer: the print composer will be deleted from the project.
 
-With the Composer Manager, it's also possible to create new print composers as an empty composer or from a saved template. By default, |qg| will look for templates in user directory (~/.qgis2/composer_template) or application's one (appDir/composer_templates). |qg| will retrieve all the available templates and propose them in the combobox. The selected template will be used to create a new composer when clicking on :guilabel:`Add` button. 
+With the Composer Manager, it's also possible to create new print composers as an empty composer or from a saved template. By default, |qg| will look for templates in user directory (:file:`~/.qgis2/composer_templates`) or application's one (:file:`appDir/composer_templates`). |qg| will retrieve all the available templates and propose them in the combobox. The selected template will be used to create a new composer when clicking on :guilabel:`Add` button.
```
Comment (yjacolin) [inline source/docs/user_manual/print_composer/managing_composer.rst]: incorrect line size (80 caracters)

Comment (DelazJ) [inline source/docs/user_manual/print_composer/managing_composer.rst]: Thanks. Will need to be more vigilant on this in the future ...

**Pull request merged**